ci/kind/kind-setup.sh

```diff
@@ -316,6 +316,7 @@ function configure_vlan_subnets {
     # Our e2e tests require external network access for regular Pods, so we need to add a custom SNAT rule.
     docker_run_with_host_net iptables -t nat -A POSTROUTING ! -o $bridge_interface -s 10.244.0.0/16 -m set ! --match-set excluded_subnets dst -j MASQUERADE
   fi
+  docker_run_with_host_net iptables -t nat -vnL
```

Comment:
May I know why this line was added? I didn’t find it in the 2.2 or main branch.

Reply:
Hi @XinShuYang , i added this to debug the nat iptables, as i can see there is no change in iptables rules but ipam tests still failed. will remove it once tests passed.

Reply:
@KMAnju-2021 Is this PR ready for review now? The Kind IPAM check is still failing.

Reply:
IPAM checks failing with every recent PR:https://github.com/antrea-io/antrea/actions/runs/14985050956/job/42097437935?pr=7164,  with main as well as 2.3, looking into it.